turbine/src/broadcast_stage/broadcast_duplicates_run.rs

```diff
@@ -176,7 +176,7 @@ impl BroadcastRun for BroadcastDuplicatesRun {
             None, // chained_merkle_root
```

Comment:
can we use `broadcast_utils::get_chained_merkle_root_from_parent(` similar to the other stages? or saving that for a separate pr?

Reply:
That is done in https://github.com/solana-labs/solana/pull/35058
But it is failing tests. So I extracted this out for now until the other one is debugged.

Reply:
the test is probably relying on being able to send a different last shred to different nodes and creating a separate valid version of the block. I'm assuming that no longer works with merkle shreds haha.

Reply:
can take a look at the test in the AM, this change looks fine to merge